nextcord/permissions.py

```diff
@@ -334,7 +334,11 @@ def view_channel(self) -> int:
 
     @flag_value
     def send_messages(self) -> int:
-        """:class:`bool`: Returns ``True`` if a user can send messages from all or specific text channels."""
+        """:class:`bool`: Returns ``True`` if a user can send messages from all or specific text channels.
+
+        .. note::
```

Comment:
```
        """:class:`bool`: Returns ``True`` if a user can moderate members (add timeouts).
        Referred to as ``Timeout Members`` in the discord client.
        .. versionadded:: 2.0
```

This does not use a note, I don't think these should be notes either.

Reply:
Noted. (pun intended :)) But on that note (haha again), should this be on the same line or be on a new line? As in `Returns ... [note here]` or 
```
Returns ...
[note here]
```

Reply:
Personal choice I'd say, doesn't really change much